src/Compute/Compute/Generated/CapacityReservation/NewAzCapacityReservationGroupCommand.cs

```diff
@@ -70,6 +70,10 @@ public class NewAzureCapacityReservationGroup : ComputeAutomationBaseCmdlet
             Mandatory = false)]
         public string[] Zone { get; set; }
 
+        [Parameter(
+            Mandatory = false)]
```

Comment:
Add in ValueFromPipelineByPropertyName  and HelpMessage. The helpMessage should match the description on the help file.

Reply:
ValueFromPipelineByPropertyName default is false, added help.